app.py

```diff
@@ -38,11 +47,13 @@ def index():
     @login_required
     def add_todo():
         if current_user.is_active == True:
-            if current_user.role == 'writer' or current_user == 'admin':
+            if current_user.role == 'writer' or current_user.role == 'admin':
                 mongo.create_new_item(request.form.get('title'))
+                app.logger.info("User %s added Todo item '%s'", current_user.name, request.form.get('title'))
```

Comment:
Would be nice to log the created item's id too, since that's what you log elsewhere (definitely optional)